libr/bin/format/mach0/mach0.c

```diff
@@ -3395,13 +3409,18 @@ static void parse_relocation_info(struct MACH0_(obj_t) *bin, RSkipList *relocs,
 		return;
 	}
 
+	const int limit = 4; // bf->rbin->limit;
```

Comment:
go back to using bf->rbin->limit?

Reply:
that was for testing, but yeah, bf or rbin are not accessible in here